src/elm/Main.elm

```diff
@@ -58,6 +61,9 @@ init flags =
       , viewportHeightWidth = ( 800, 800 )
       , chartHovering = []
       , terminalState = { input = "", history = [] }
+      , pile1 = Pile.init (View.Posts.posts Data.Section1 (Data.decodedContent flags |> (\c -> c.posts)))
```

Comment:
You are right, this is weird pattern having html in the model. We can refactor at some point.

Reply:
Is there some obvious state we can store to feed the view rather than the Html?